earth_enterprise/rpms/build.gradle

```diff
@@ -0,0 +1,131 @@
+/*
+    See documentation on the osPackage plugin here: https://github.com/nebula-plugins/gradle-ospackage-plugin
+
+    This packaging script depends on the output of the main scons "stage_install" target,
+    which copies build artifacts, binaries and documents in to the /tmp/fusion_os_install path.
+ */
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.netflix.nebula:gradle-ospackage-plugin:4.4.2"
+    }
+}
+
+apply plugin: "nebula.ospackage"
+
+def env = System.getenv()
+def staged_install_path = '/tmp/fusion_os_install'
```

Comment:
Oh, I see.  You're requiring the developer build with the proper options and `stage_install`.

I have a task that runs the `scons` commands beforehand.  But, what's here is enough for the spike.